sema/errors.go

```diff
@@ -4739,6 +5439,32 @@ func (e *DefaultDestroyInvalidParameterError) Error() string {
 	return fmt.Sprintf("`%s` is not a valid parameter type for a default destroy event", e.ParamType.QualifiedString())
 }
 
+func (e *DefaultDestroyInvalidParameterError) SecondaryError() string {
+	return "default destroy events only support storable types as parameters. Consider using a concrete type instead of `Any`"
```

Comment:
The `Any` type used in the example error message is just a placeholder, the error might mention other types. Maybe just remove the second sentence:
```suggestion
	return "default destroy events only support storable types as parameters"
```